src/components/PopupWithForm.js

```diff
@@ -27,6 +29,14 @@ export default class PopupWithForm extends Popup {
     return this._popupForm;
   }
 
+  loadingState(isLoading, loadingText = "Saving...") {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do. **Please, correct it everywhere in the code**

`setLoadingState`